014_kf_L91.py

```
@@ -0,0 +1,316 @@
+    else: 
+        exit()
+
+# Custom path chooser
+# {fact rule=code-injection@v1.0 defects=1}
+def mover(out_file):
+    move= input(ask+"Move to a custom path?(y/n) > "+green)
+    if move=="y":
+        mpath=input(ask+"Enter the path > "+ green)
+        if os.path.exists(mpath):
+            # {fact rule=os-command-injection@v1.0 defects=0}
+            os.system(f'''mv -f "{out_file}" "{mpath}" ''')
```

Ruff (0.14.5)

Starting a process with a shell, possible injection detected

(S605)